x/dualstaking/types/errors.go

```diff
@@ -13,4 +13,5 @@ var (
 	ErrBadDelegationAmount       = sdkerrors.Register(ModuleName, 1003, "invalid delegation amount")
 	ErrUnbondingInProgress       = sdkerrors.Register(ModuleName, 1004, "unbonding already exists (same block)")
 	ErrCalculatingProviderReward = sdkerrors.Register(ModuleName, 1005, "provider reward calculation failed")
+	ErrWrongDenom                = sdkerrors.Register(ModuleName, 1006, "amount has wrong denomanator")
```

Comment:
denominator

Reply:
create an error message in the common module (or utils) and use a single message for all modules (I can see there is a different message in the plans module)

Reply:
Fixed in 1697968c86cc5c83a771ee9803e811b3a1b1a9dd